src/styles/blocks/_variables.scss

```
@@ -0,0 +1,11 @@
+$nav_color: #060b35;
+$distance_between: 48px;
+
+$time_animation: 0.3s;
+$nav_color_active: #00acdc;
+$color_text_active: #34568b;
```

it is better to write similar properties next to each other
```suggestion
$distance_between: 48px;
$time_animation: 0.3s;

$nav_color: #060b35;
$nav_color_active: #00acdc;
$color_text_active: #34568b;
```